### PR TITLE
Remove dupe and

### LINKED ISSUE #2771: Double wording on front page
In this [page](https://ocaml.org/industrial-users) you have double "and" in the text "OCaml powers mission-critical systems for companies and and helps them achieve their goals."

### PR DESCRIPTION
**Pull request by cuihtlauac**
Fix #2771 

Note sure we should fix stuff from data/planet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,28 +9,26 @@ This repository contains the sources of the OCaml website. It is served at https
...
-  the look and feel of the webpage, as well as make it easier to navigate, and more
+  the look and feel of the webpage, as well as make it easier to navigate and more
   accessible (particularly on mobile devices).
 
 - **Separation of data editing from HTML/CSS generation:** The data used in the website
-  is stored in Yaml or Markdown so users can easily edit it and contribute to the website.
-  We generate OCaml code from this data to serve the site content and and Ocurrent to
+  is stored in YAML or Markdown, so users can easily edit it and contribute to the website.
+  We generate OCaml code from this data to serve the site content and OCurrent to
   automate the data pipelines. All the data used in the site can be found in [`./data`](./data).
```
Comment (christinerose) [inline README.md]:
```suggestion
  We generate OCaml code from this data to serve the site content and for OCurrent to
  automate the data pipelines. All the data used in the site can be found in [`./data`](./data).
```
Does "we generate OCaml code" refer to both "serve the site content" and "OCurrent to automate" ?? Or is OCurrent automating separate from this code generation?

File: README.md [hunk 2]
```diff
@@ -73,43 +72,37 @@ In particular:
...
-
-We’d also like to thank the major funders who supported work on revamping the website: grants from the Tezos Foundation and Jane Street facilitated the bulk of the work. Thank you, and if anyone else wishes to help support it on an ongoing basis then donations to the OCaml Software Foundation and grants to the maintenance teams mentioned above are always welcomed.
+We’d also like to thank the major funders who supported work on revamping the website. Grants from the Tezos Foundation and Jane Street facilitated the bulk of the work. Thank you! And if anyone else wishes to help support it on an ongoing basis, then donations to the OCaml Software Foundation and grants to the maintenance teams mentioned above are always welcomed.
 
 ## Contributing
 
-We'd love your help improving ocaml.org!
+We'd love your help improving OCaml.org!
 
 See our contributing guide in [`CONTRIBUTING.md`](./CONTRIBUTING.md)
```
Comment (christinerose) [inline README.md:96]: This is the second time in the README this is mentioned and linked. Do you want it there twice?

Comment (cuihtlauac) [reply README.md:96]: You are right @christinerose, the first occurrence should be removed.

Comment (cuihtlauac) [reply README.md:96]: Fixed

File: README.md (hunk 1 above)
Comment (cuihtlauac) [reply README.md]: I've rewritten this sentence

**Comment by sabine** [comment]
We shouldn't edit the OCaml Planet entries, indeed.

**Review by sabine — APPROVED** [review]

**Pull request merged**